tests/src/python/test_qgsanalysis.py

```diff
@@ -45,7 +36,7 @@ def tearDown(self):
         """Run after each test."""
         pass
 
-    #@expectedFailure
+    #@unittest.expectedFailure
```

Comment:
Just wondering- have you retested any of these expected failures? I wonder if any can be unmarked as failures now